tools/bin/check_images_exist.sh

```diff
@@ -1,78 +1,133 @@
 #!/usr/bin/env bash
 
-set -e
+# ------------- Import some defaults for the shell
+
+# Source shell defaults
+# $0 is the currently running program (this file)
+this_file_directory=$(dirname $0)
```

Comment:
do we want to use the `assert_root` function from lib.sh (like we do for some other scripts)? https://github.com/airbytehq/airbyte/blob/289a0654018d133a01651722d17a9f33cce7f40d/tools/lib/lib.sh#L6-L8

btw looks like we source lib.sh on line 22 of this script

Reply:
I'm on mobile so no link, but it's check root is called from main()